client/src/components/tools/player/ah.jsx

```diff
@@ -19,7 +19,7 @@ const formatItem = (itemname, stacksize) => {
             string.length
           )}`;
         })
-        .join(' ') + (stacksize === '1' ? '' : ' x' + stacksize)}
+        .join(' ') + (parseInt(stacksize) === 1 ? '' : ' x' + stacksize)}
```

Comment:
I'd reverse the logic here a bit to have it be more resilient against other failure-states, like if stacksize can't be parsed as an int.
So something like:
```typescript
parseInt(stacksize) > 1 ? ` x${stacksize}` : '')
```

Reply:
Done - you also made me realize there's the need for another parseInt slightly higher up.